src/sass/partials/components/_content-card.scss

```diff
@@ -0,0 +1,30 @@
+.card {
```

Comment:
Is `.card` generic on purpose? Would this styling be _only_ for the content card, or any card?